### ISSUE TITLE
Add an example integrated with Spring Boot

### ISSUE THREAD

**Issue by jabrena** [pivovarit/parallel-collectors#134]
**Is your feature request related to a problem? Please describe.**

I would like to add an example integrated in a Spring Boot Application

**Describe the solution you'd like**

I would like to add an Example explained as Use Case with the library.

Many thanks in advance.

**Comment by pivovarit** [comment]
I don't think that addition of a Spring Boot helps us here in any way - let me explain.

Spring Boot is a framework that takes our code and puts in some form of a frame. In this particular scenario, this library doesn't even get close to the boundaries of that frame.

Having said that, we could take all existing examples, put them in a Spring Boot application but those would be still exactly same examples - just running in a Spring Boot context - there's no specific integration to be showcased. 

What's more, you could replace your standard `Collectors.to*` with those and your code will work regardless of the framework used (I'm ignoring resources usage on purpose to make that analogy simpler). Does that answer satisfy you?
